LogsHandler/src/main/java/boot/ServerModule.java

```diff
@@ -0,0 +1,79 @@
+package boot;
+
+import Producer.KafkaPublisher;
+import com.google.inject.AbstractModule;
+import javax.inject.Inject;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import java.util.Properties;
+
+import com.google.inject.Provides;
+import io.logz.guice.jersey.JerseyModule;
+import io.logz.guice.jersey.configuration.JerseyConfiguration;
+import org.apache.http.HttpHost;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.LongSerializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.glassfish.jersey.client.ClientConfig;
+import org.glassfish.jersey.jackson.internal.jackson.jaxrs.json.JacksonJsonProvider;
+
+import util.InfraUtil;
+
+
+public class ServerModule extends AbstractModule {
+    JerseyConfiguration configuration ;
+    ServerConfiguration serverConfiguration;
+    @Inject
+    public ServerModule(){
+        try {
+            serverConfiguration = InfraUtil.load(ServerConfiguration.configFilePath, ServerConfiguration.class);
+        }catch(Exception e){
+            System.out.println(e.getMessage());
+            try {
+                serverConfiguration = InfraUtil.load(ServerConfiguration.configFilePath, ServerConfiguration.class);
+            }catch(Exception exception){
+                System.out.println(exception.getMessage());
+            }
+        }
+        configuration = JerseyConfiguration.builder()
+                .addPackage("boot")
+                .addPort(serverConfiguration.getPort())
+                .build();
+    }
+    @Override
+    protected void configure(){
+        binder().requireExplicitBindings();
+        install(new JerseyModule(configuration));
+        bind(EntryPoint.class);
+        bind(ElasticSearchHandler.class);
+        bind(KafkaPublisher.class);
+    }
+
+    @Provides
+    public Client providesClient() {
```

Comment:
I see you did it for tests
We tend not to use Dependency Injection in tests. Instead, Let's create a simple `WebTarget` as a client for tests (ask the other guys for reference)